quickstart/faqs.md

```diff
@@ -79,3 +79,12 @@ Mayastor has been designed so as to be able to leverage the peformance capabilit
 ## Does the supportability tool expose sensitive data?
 
 The supportability tool generates support bundles, which are used for debugging purposes. These bundles are created in response to the user's invocation of the tool and can be transmitted only by the user. To view the list of collected information, visit the [supportability section](https://mayastor.gitbook.io/introduction/reference/supportability#does-the-supportability-tool-expose-sensistive-data).
+
+
+## What happens when a PV with reclaim policy set to `retain` is deleted?
+
+In Kubernetes, when a PVC is created with the reclaim policy set to 'Retain', the PV bound to this PVC is not deleted even if the PVC is deleted. One can manually delete the PV by issuing the command "kubectl delete pv ", however the underlying storage resources could be left behind as the CSI volume provisioner (external provisioner) is not aware of this. To resolve this issue of dangling storage objects, Mayastor has introduced a PV garbage collector. This PV garbage collector is deployed as a part of the Mayastor CSI controller-plugin.
+
+  ### How does the PV garbage collector work?
+
+   The PV garbage collector deploys a watcher component, which subscribes to the Kubernetes Persistent Volume deletion events. When a PV is deleted, an event is generated by the Kubernetes API server and is received by this component. Upon a successful validation of this event, the garbage collector deletes the corresponding Mayastor volume objects.
```

Comment:
last word should be resources instead of objects, lgtm

Reply:
Updated.